docs/Tutorial_AB_Joins.ipynb

```diff
@@ -289,6 +289,25 @@
     "print(f'The motif is located at index {vanilla_ice_motif_index} of \"Ice Ice Baby\"')"
```

Comment:
<p>A minor typo in the inline comment provided in the code cell. </p><p> </p>

<pre class="ql-syntax" spellcheck="false">Replace matric with matrix
</pre><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/972/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>